tests/run_tests.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
```

Comment:
The [unofficial bash strict mode](http://redsymbol.net/articles/unofficial-bash-strict-mode/) would be safer, just in case:

```
    #!/bin/bash
    set -euo pipefail
    IFS=$'\n\t'
```

Reply:
That would also imply replacing `$1` by `${1:-}` to avoid referencing unbound variables all over the shell script. You may want to check [shellcheck](https://www.shellcheck.net)?